cypress/integration/advanced-functionalities/server/Advanced_functionalities.spec.js

```diff
@@ -0,0 +1,106 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+const newEmail = Cypress.env('user').newEmail
+const newPassword = Cypress.env('user').newPassword
+
+describe('Test server advanced functionalities ', () => {
+  beforeEach(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ baseURL })
+      cy.get('[data-test-id="input-email"]').type(newEmail)
+      cy.get('[data-test-id="input-password"]').type(newPassword)
+      cy.get('[data-test-id="button-login-and-remember"]').click()
+    }
+  })
+
+  context('Testing your profile tab', () => {
+    it('should be able to change the name', () => {
+      cy.visit(baseURL + '/auth/login')
+      cy.get('[data-test-id="input-name"]').type(' edited')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We check if the success toast appears
+      cy.get('.border-primary')
+      cy.get('[data-test-id="input-name"]').should('have.value', 'Permissions account edited')
+    })
+
+    it('should be able to change the handle', () => {
+      cy.get('[data-test-id="input-handle"]').type('_edited')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We check if the success toast appears
+      cy.get('.border-primary')
+      cy.get('[data-test-id="input-handle"]').should('have.value', 'permissions_account_edited')
+    })
+
+    it('should be able to change the preferred language', () => {
+      cy.get('[data-test-id="select-language"]').select('German (German)')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We check if the success toast appears
+      cy.get('.border-primary')
+      cy.get('[data-test-id="select-language"]').should('exist', 'German (German)')
+    })
+  })
```

Comment:
instead of repeating the same code: change something -> save -> check if okay, can we just update the input values -> save -> check them?